public/secure/reset.js

```diff
@@ -23,7 +24,7 @@ function performApiRequest(url, data) {
     //     Currently API sends only 400 `{"message": "error"}`
     if (response.status !== 200) {
       throw new WrongInputError(
-        'Looks like you typed in wrong password. Please try it again!'
+        'Password reset error. Please request reset password again!'
```

Comment:
The same here about the exclamation mark.

I would slightly change the message but I do not know what exactly it should be. Leave it as is, I will review it with @nancypontika when have time.